lib/loc.ml

```diff
@@ -40,8 +40,12 @@ let mergeable loc1 loc2 =
   loc1.fname = loc2.fname
 
 let merge loc1 loc2 =
+  if loc1 = make_loc (0,0) then loc2 else
```

Comment:
Using pervasive equality here is looking for trouble. Please write a `is_dummy` function instead, in addition it will make the intent clear.

Reply:
I knew you would say that. This is probably marginal here, but ok, done.